<?xml version="1.0" encoding="utf-8"?>
<launch>
    <node pkg="odom_visualization" type="gazebo_state" name="gazebo_state" output="screen">
        <param name="model_name" value="iris"/>
        <param name="model_frame" value="base_link"/>
        <param name="target_frame" value="world"/>
    </node>
</launch>